packages/dotcom-shared/src/tla-schema/TldrawAppUser.ts

```diff
@@ -52,18 +53,33 @@ export const tldrawAppUserValidator: T.Validator<TldrawAppUser> = T.model(
 		}),
 		flags: T.object({
 			placeholder_feature_flag: T.boolean,
+			publish: T.boolean,
 		}),
 	})
 )
 
 /** @public */
 export const tldrawAppUserVersions = createMigrationIds('com.tldraw.user', {} as const)
 
+const Versions = createMigrationIds('com.tldraw-app.user', {
+	AddPublishFlag: 1,
+} as const)
+
 /** @public */
 export const tldrawAppUserMigrations = createRecordMigrationSequence({
 	sequenceId: 'com.tldraw-app.user',
 	recordType: 'user',
-	sequence: [],
+	sequence: [
+		{
+			id: Versions.AddPublishFlag,
```

Comment:
Feels a bit strange what we had to add a migration to introduce a feature flag. Maybe our flags could just be a map for string to bool?